_releaser/aws.go

```diff
@@ -100,15 +100,15 @@ type AwsCloudfrontUpdateCmd struct {
 	Function              string `kong:"name='lambda-function',env='AWS_LAMBDA_FUNCTION'"`
 	FunctionFile          string `kong:"name='lambda-function-file',env='AWS_LAMBDA_FUNCTION_FILE'"`
 	CloudfrontID          string `kong:"name='cloudfront-id',env='AWS_CLOUDFRONT_ID'"`
-	RedirectsJSON         string `kong:"name='redirects-json',env='REDIRECTS_JSON'"`
-	RedirectsPrefixesJSON string `kong:"name='redirects-prefixes-json',env='REDIRECTS_PREFIXES_JSON'"`
+	RedirectsFile         string `kong:"name='redirects-json',env='REDIRECTS_FILE'"`
```

Comment:
https://github.com/alecthomas/kong?tab=readme-ov-file#custom-named-decoders

```suggestion
	RedirectsFile         string `kong:"name='redirects-json',type=fileContent,env='REDIRECTS_FILE'"`
```

You can then remove `os.ReadFile`

Reply:
Also don't forget to rename the flag `redirects-json` > `redirects-file`